samples/wifi/raw_tx_packet/README.rst

```diff
@@ -0,0 +1,163 @@
+.. _wifi_raw_tx_packet_sample:
+
+Wi-Fi: Raw TX packet
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Raw TX packet sample demonstrates how to transmit raw TX packets.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+
+The sample demonstrates how to transmit raw TX packets in both connected Station and non-connected Station modes of operation.
+The sample provides the option to transmit raw TX packets either continuously or in a fixed number of packets.
```

Comment:
can this be made a bulleted point such as 

The sample provides the option to transmit raw TX packets as described below:
1) Continuous packet transmission. 
2) Fixed number of packet transmit.